Assets/Scripts/OriginalPlayerCharcter.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class OriginalPlayerCharcter : MonoBehaviour
+{
+    //private = camelCase public = PascalCase
```

Comment:
Remove this comment before sumbitting. I know it's useful for you, just not needed in the build